x-pack/test_serverless/functional/services/svl_reporting.ts

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
This file isn't imported from anywhere (which is good) and can be deleted

Reply:
Wait I think it's being used in x-pack/test_serverless/shared/services/index.ts...?